ledger/ledger-api-client/src/test/suite/scala/com/digitalasset/ledger/client/services/commands/CommandTrackerFlowTest.scala

```diff
@@ -300,16 +280,51 @@ class CommandTrackerFlowTest
       }
 
       "timeout the command when the MRT passes" in {
```

Comment:
Should we update this description as well? I.e., I'm not sure if MRT makes sense in this context anymore.

Reply:
Done for the next PR.